tools/refguide_check.py

```diff
@@ -331,6 +420,19 @@ def check_items(all_dict, names, deprecated, others, module_name, dots=True):
 
 
 def validate_rst_syntax(text, name, dots=True):
+    """
+    Validates the doc string text of a module `name`
```

Comment:
not module, this is for RST documentation. So something like "Validates the doc strings in a snippet of documentation `text` from file `name`

Reply:
Ok